packages/features/users/components/UserTable/BulkActions/TeamList.tsx

```diff
@@ -0,0 +1,133 @@
+  const mutation = trpc.viewer.organizations.bulkAddToTeams.useMutation({
+    onError: (error) => {
+      showToast(error.message, "error");
+    },
+    onSuccess: (res) => {
+      showToast(
+        `${res.invitedTotalUsers} Users invited to ${Array.from(selectedValues).length} teams`,
+        "success"
+      );
+      // Optimistically update the data from query trpc cache listMembers
+      // We may need to set this data instread of invalidating. Will see how performance handles it
+      utils.viewer.organizations.listMembers.invalidate();
+
+      // Clear the selected values
+      setSelectedValues(new Set());
+      table.toggleAllRowsSelected(false);
+    },
+  });
+
+  const { t } = useLocale();
+
+  // Add a value to the set
+  const addValue = (value: number) => {
+    const updatedSet = new Set(selectedValues);
+    updatedSet.add(value);
+    setSelectedValues(updatedSet);
+  };
+
+  // Remove a value from the set
+  const removeValue = (value: number) => {
+    const updatedSet = new Set(selectedValues);
+    updatedSet.delete(value);
+    setSelectedValues(updatedSet);
+  };
+
+  return (
+    <>
+      <Popover>
+        <PopoverTrigger asChild>
+          <Button StartIcon={Users}>{t("add_to_team")}</Button>
+        </PopoverTrigger>
+        {/* We dont really use shadows much - but its needed here  */}
+        <PopoverContent className="w-[200px] p-0 shadow-md" align="start" sideOffset={12}>
+          <Command>
+            <CommandInput placeholder={t("search")} />
+            <CommandList>
+              <CommandEmpty>No results found.</CommandEmpty>
+              <CommandGroup>
+                {teams &&
+                  teams.map((option) => {
+                    const isSelected = selectedValues.has(option.id);
+                    return (
+                      <CommandItem
+                        key={option.id}
+                        onSelect={() => {
+                          console.log("onSelect", isSelected);
+                          if (!isSelected) {
+                            addValue(option.id);
+                          } else {
+                            removeValue(option.id);
+                          }
+                        }}>
+                        <span>{option.name}</span>
+                        <div
+                          className={classNames(
+                            "border-subtle ml-auto flex h-4 w-4 items-center justify-center rounded-sm border",
+                            isSelected ? "text-emphasis" : "opacity-50 [&_svg]:invisible"
+                          )}>
+                          <Check className={classNames("h-4 w-4")} />
+                        </div>
+                      </CommandItem>
+                    );
+                  })}
+              </CommandGroup>
+              <>
+                <CommandSeparator />
+                <CommandGroup>
+                  <div className="mb-1.5 flex w-full">
+                    <Button
```

Comment:
This button is not visible when we do a search for team.
![image](https://github.com/calcom/cal.com/assets/1780212/9420f486-d3f5-4b26-a68b-d2c938742744)